hack/bumper.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -e
+
+# package bump helper
+# make sure you already have a local repo with remote upstream, and origin branches
+# no untracked files allowed on folder
+# it does reset --hard so beware, also pushes to save time
+
+# ./hack/bumper.sh CVE-2021-38561 golang.org/x/text@v0.3.7 release-0.89
+
+# to skip semver check (for example in case package is dropped from go.mod after updating)
+# export SKIP=true
+# optional: to update go lang level use for example
+# export DESIRED_VERSION=1.18
+
+SKIP=${SKIP:-false}
+
+if [ $# -ne 3 ]; then
+    echo "Syntax: $0 <CVE> <TARGET_PACK> <BR>"
+    exit 1
+fi
+
+CVE=$1
+TARGET_PACK=$2
+BR=$3
+
+PACK="${TARGET_PACK/@*}"
+[ "$(git rev-parse --abbrev-ref HEAD)" != "$BR" ] && git checkout "$BR"
+git reset --hard upstream/"$(git symbolic-ref --short HEAD)" # || true # try removing the suffix, if it doesnt work try git fetch upstream first
+git pull upstream "$(git symbolic-ref --short HEAD)"
+git push -u origin
+go mod edit -dropreplace="${PACK}"
+go mod edit -require="${TARGET_PACK}"
+
+current_version=$(awk '$1 == "go" {print $2}' go.mod)
+desired_version=${DESIRED_VERSION:-$current_version}
+if [[ "$(printf "%s\n" "$current_version" "$desired_version" | sort -V | head -n1)" != "$desired_version" ]]; then
+  echo "Updating Go version in go.mod to $desired_version"
+  go mod edit -go=$desired_version
+else
+  echo "Go version in go.mod is already $desired_version or later. No update needed."
+fi
+
+if grep -q '^vendor:' Makefile; then
+    rm -rf build/_output/
+    make vendor
+else
+    go mod tidy
```

Comment:
should we do `-compat=${desired_version}`?

Reply:
i would say not yet
because it was meant for OVS, which doesnt do it as well, so they should be the same
https://github.com/k8snetworkplumbingwg/ovs-cni/blob/main/Makefile#L91

btw what we can do for OVS, is to add `make format` if exists
https://github.com/k8snetworkplumbingwg/ovs-cni/blob/main/Makefile#L55
which does fmt / vet

Reply:
I would say on long term align, and on short term just add all options? wdyt?

Reply:
i would suggest please to start simple, as is
and adapt OVS / make sure all the other repos are aligned, and finally change this one, wdyt?

Reply:
yeah I'm good with that